pkg/agent/cniserver/pod_configuration.go

```diff
@@ -136,7 +140,7 @@ func configureContainerAddr(netns ns.NetNS, containerInterface *current.Interfac
 		// and the gratuitous ARP could be sent out after the Openflow entries are installed. Using another goroutine
 		// ensures the processing of CNI ADD request is not blocked.
 		go func() {
-			netns.Do(func(containerNs ns.NetNS) error {
+			_ = netns.Do(func(containerNs ns.NetNS) error {
```

Comment:
can we avoid doing this? I don't think it improves style and that's not flagged by any of the linters we run in CI.

Reply:
to me, I'd like to think if a method return an error, it needs to be handled, even though it is don't care in this particular use case. It tell user in the context of usage that error does not happen. 

I could not find a good/authentic source to back my reasoning, hope this provide some similarly thought process https://travix.io/unhandled-errors-in-go-3f341f2704dd; but feel free to prove I am wrong.

Reply:
I have the same feeling with @antoninbas, `_ = XXX` and code like `defer func() { _ = f.Close() }()` looks not helpful and a little overkill, especially I have never seen the latter in other projects. Ignoring errors is quite common even in go src code, for example: 
https://github.com/golang/go/blob/cc6a8bd0d7f782c31e1a35793b4e1253c6716ad5/src/net/http/httputil/dump.go#L115
https://github.com/golang/go/blob/cc6a8bd0d7f782c31e1a35793b4e1253c6716ad5/src/net/http/httputil/dump.go#L137

Reply:
Even code in effective go introducing `defer` doesn't do it: https://golang.org/doc/effective_go.html#defer.

Reply:
If you use the Goland IDE, you can disable the "Unhandled Error" warnings. Unfortunately there doesn't seem to be a way to only disable the warning for `defer` statements.